docs/source/settings.rst

```diff
@@ -185,9 +185,10 @@ graceful_timeout
 
 Timeout for graceful workers restart.
 
-Generally set to thirty seconds. How max time worker can handle
-request after got restart signal. If the time is up worker will
-be force killed.
+Generally set to thirty seconds. After receiving a restart signal,
```

Comment:
I would remove this first sentence. It repeats information that is already above, in the default value.